apps/studio/src/components/PageEditor/constants.ts

```diff
@@ -159,3 +172,114 @@ export const DEFAULT_BLOCKS: Record<
+  },
+  callout: {
+    label: "Callout",
+    icon: BiSolidQuoteAltLeft,
+    description: "Bring attention to important information",
+    usageText:
+      "Callouts are great for highlighting information such as updates. We recommend not overusing the callouts.",
+  },
+  keystatistics: {
+    label: "Statistics",
+    icon: BiHash,
+    description: "Display KPIs or key statistics for your agency",
+    usageText:
+      "Do you have metrics to show the public? Designed to be bold, this block supports up to four numbers with labels.",
+  },
+  infobar: {
+    label: "Text with CTA",
+    icon: ContentpicIcon,
+    description: "Add a strong call-to-action",
+    usageText:
+      "Use this block to highlight key initatives on your homepage. It supports up to two buttons.",
+  },
+  contentpic: {
+    label: "Contentpic",
+    icon: ContentpicIcon,
+    description: "Put an image and text side-by-side",
+    usageText:
+      "Use this block to juxtapose text next to a smaller image than usual, such as introducing a committee member along with their headshot.",
+  },
+  infopic: {
+    label: "Text with image",
+    icon: ContentpicIcon,
+    description: "Place an image with a text and call-to-action",
+    usageText: "This block comes with a button.",
+  },
+  accordion: {
+    label: "Accordion",
+    icon: BiChevronDown,
+    description: "Display content in expandable accordions",
+    usageText:
+      "Accordions hide details by default, so they are great for content that isn't relevant to every reader.",
+  },
+  infocards: {
+    label: "Cards",
+    icon: BiCard,
+    description: `Link information in "cards" with or without images`,
+    usageText: "This block supports up to six cards.",
+  },
+  infocols: {
+    label: "Columns of text",
+    icon: BiColumns,
+    description: "Show important links in multiple columns",
+    usageText: "This block supports up to six links.",
+  },
+  iframe: {
+    label: "Embed",
+    icon: BiColumns,
+    description: "Embed a video or other content",
+    usageText: "This block supports embedding content from other websites.",
+  },
+}
+
+type AllowedBlockSections = {
+  label: string
+  types: IsomerComponent["type"][]
+}[]
+
+export const ARTICLE_ALLOWED_BLOCKS: AllowedBlockSections = [
+  { label: "Basic building blocks", types: ["prose", "image", "callout"] },
+]
+
+export const CONTENT_ALLOWED_BLOCKS: AllowedBlockSections = [
+  { label: "Basic building blocks", types: ["prose", "image", "callout"] },
+  {
+    label: "Organise complex content",
+    types: ["contentpic", "infocards", "accordion", "infocols", "iframe"],
+  },
+]
+export const HOMEPAGE_ALLOWED_BLOCKS: AllowedBlockSections = [
+  {
+    label: "Highlight important information",
+    types: ["keystatistics", "infobar"],
+  },
+  {
+    label: "Organise complex content",
+    types: ["contentpic", "infocards", "infocols", "iframe"],
```

Comment:
issue: should be infopic not contentpic

```suggestion
    types: ["infopic", "infocards", "infocols", "iframe"],
```

Reply:
u right